saleor/graphql/order/tests/queries/test_order_with_filter.py

```diff
@@ -1360,3 +1360,62 @@ def test_order_query_with_filter_by_empty_list(
     # then
     content = get_graphql_content(response)
     assert content["data"]["orders"]["totalCount"] == len(orders_from_checkout)
+
+
+def test_order_query_with_filter_checkout_tokens(
+    orders_query_with_filter,
+    staff_api_client,
+    permission_group_manage_orders,
+    orders,
+    checkout,
+    channel_USD,
+):
+    assert not any([order.checkout_token for order in orders])
+    order = orders[0]
+    order.checkout_token = checkout.pk
+    order.save()
```

Comment:
You are setting the value only on one object, IMO few orders should have `checkout_token` set to check if the filter is working properly.